roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -186,12 +186,11 @@ kiali_defaults:
         use_kiali_token: false
         username: ""
       enabled: true
-      health_check_url: ""
       in_cluster_url: ""
       is_core: false
       namespace_selector: true
       url: ""
-      #use_grpc:
+      use_grpc: true
```

Comment:
In the comment above on line 641 it says,"#If not specified, it will be false if deployed within a Maistra/OSSM+OpenShift environment, true otherwise."

I think this is still true but I would like to confirm

Reply:
I answered my own question - the comment is correct. See: https://github.com/kiali/kiali-operator/blob/v1.38.0/roles/default/kiali-deploy/tasks/main.yml#L324-L326

Reply:
Because this is now defaulting to true, we should update the kiali_cr.yaml example file to reflect this - see line 658 of kiali_cr.yaml